cmd/pbstatuser/main.go

```diff
@@ -55,6 +56,10 @@ var (
 	senderWG     = sync.WaitGroup{}
 )
 
+func init() {
+	random.SeedGlobal()
+}
```

Comment:
This is needed in all commands, we were missing this.

With Go 1.20 we can remove this. But we are still on 1.18 in prod.